contents/founders/actual-marketing-budget.md

```diff
@@ -0,0 +1,97 @@
+
+How much does a typical startup [like PostHog](/about) spend on marketing? And why doesn’t anyone talk about it?
+
+“But Charles, you naive idiot” asked nobody, “surely competitors will use this information against you?”
+
+Um, sure I guess? But especially if you’re into product-led growth (like we are), how you spend marketing $ seems pretty far down the list of competitive advantages. We already [tell everyone](/roadmap) what we’re going to build next, and that doesn’t seem to have hurt us so far.
+
+By sharing some of this info, I’m hoping it’ll help others as a data point. **I am not saying this is the best way to spend your marketing budget.** I have no idea! That’s the point!! But I’m hoping _maybe_ someone will see this and tell us how we could do better...
+
+Let’s get into it. 
+
+## Show me the money
+
+This is a summary of what a typical monthly marketing spend looks like for us:
+
+- **People: $72,000 per month**
+  - 3 content marketers
+  - 1 product marketer
+  - 1 graphic designer
+  - 1 frontend developer
+  - 0.5 exec who mostly gets in the way
+- **Not people: $73,000 per month**
+  - Tools & subscriptions: $2,000
+  - Paid ads: $40,000
+  - Agency to manage the paid ads: $5,000
+  - Sponsorships:
+  - Newsletters: $15,000
+  - Influencers: $5,000
+  - Open source projects: $2,000
+  - Merch: $3,000
+  - Events: $0
+  - PR: $0
+- **Total: $144,000 per month**
+
+### Some notes
+
+- We do all of our content in-house. We have very occasionally worked with freelancers in the past, but we haven’t done this for a long time. 
+- Technically our frontend developer is not part of the marketing team ([Website & Docs](/handbook/small-teams/website-docs) is its own team at PostHog), but I appreciate at most companies your website will be considered a marketing expense. 
+- The 50-50 split is coincidental. 
+
+If you want to learn more about how we implement these things in practice at PostHog, it’s all in our [public handbook](/handbook/growth/marketing). 
+
+## Going deeper on paid ads
+
+I’ve written about [running paid ads channels](/founders/dev-marketing-paid-ads) before, but this is how that $40k split looked over the last 3 months on average (as the channel mix varies month to month):
+
+- **Conversion**
+  - Google Search - brand: 10%
+  - Google Search - generic: 50%
+- **Awareness**
+  - Twitter: 10%
+  - LinkedIn: 10%
+  - Quora: 10%
+  - Random experiments (e.g. G2, Bing): 0-10%
+
+As the barrier to implementing PostHog is reasonably high in the grand scheme of SaaS tools - you need a software product that has users, instrumenting PostHog isn’t totally trivial - we don’t use the channels listed under awareness to try and drive sign ups at all. 
```

Comment:
The middle part of this sentence makes it tough to read, would be better split up.

Reply:
+1